invmul_test.go

```diff
@@ -0,0 +1,122 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/crate-crypto/go-ipa/bandersnatch/fp"
+	"github.com/stretchr/testify/require"
+	blst "github.com/supranational/blst/bindings/go"
+)
+
+func BenchmarkFpInverse(b *testing.B) {
+	b.Run("go-ipa", func(b *testing.B) {
+		var x fp.Element
+		x.SetRandom()
+
+		b.ReportAllocs()
+		b.ResetTimer()
+		var resIPA fp.Element
+		for i := 0; i < b.N; i++ {
+			resIPA.Inverse(&x)
+		}
+	})
+	b.Run("go-blst", func(b *testing.B) {
+		var xIPA fp.Element
+		xIPA.SetRandom()
+		var xBLST blst.Scalar
+		beBytes := xIPA.Bytes()
+		xBLST.FromBEndian(beBytes[:])
```

Comment:
Just leveraging the `.SetRandom()` form `go-ipa`, and mapping that to the scalar type in `blst` for benchmarking.